app/src/main/java/io/intrepid/contest/rest/MockRestApi.java

```diff
@@ -116,8 +117,11 @@ private ContestStatusResponse getValidEntryResponseWaitingForScores() {
     private ContestWrapper getValidContestResponse() {
         Contest contest = new Contest();
         contest.setTitle(contestTitle);
+        List<Category> contestCategories = contest.getCategories();
+        for (int i = 0; i < 5; i++) {
+            contestCategories.add(new Category("Category " + i, "This is category " + i)); //temp - fixme
```

Comment:
I don't think you need the `// temp - fixme` comment here. It's a mock class, it's assumed you're just creating mock content.